features/protected-audience.yml

```diff
@@ -1,7 +1,8 @@
-draft_date: 2025-01-20
-name: Protected Audience (formerly FLEDGE)
-description: TODO
+name: Protected Audience
+description: The Protected Audience API facilitates advertisement sales by allowing sites to register users as part of an interest group or to choose which ads appear based on those interest groups, while minimizing the ability of advertisers to track specific members of the interest group. Also known as FLEDGE.
 spec: https://wicg.github.io/turtledove/
+status:
+  compute_from: api.Navigator.getInterestGroupAdAuctionData
```

Comment:
This is a bit awkward. It might be nice to unset this when Chrome 134 hits stable.

See also https://github.com/web-platform-dx/web-features/pull/2638#discussion_r1951099639